tests/skipCrossGenFiles.arm.txt

```diff
@@ -0,0 +1,2 @@
+mscorlib.dll
+System.Runtime.WindowsRuntime.UI.Xaml.dll
```

Comment:
@sdmaclea Thank you for letting me know. 
@jyoungyun @hqueue @hseok-oh Do we currently have any files that should be included here?